src/easy.c

```diff
@@ -1882,6 +1882,10 @@ do_curl_setopt_file_passthrough(CurlObject *self, int option, PyObject *obj)
     switch (option) {
     case CURLOPT_READDATA:
     case CURLOPT_WRITEDATA:
+        res = curl_easy_setopt(self->handle, CURLOPT_WRITEFUNCTION, fwrite);
```

Comment:
Reading this again it looks weird, I need to refresh my memory of stdio passthrough to figure out whether this is in fact correct.